src/parser/rogue/subtlety/modules/spells/FindWeakness.tsx

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 
-import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Analyzer from 'parser/core/Analyzer';
 import SPELLS from 'common/SPELLS';
-import SpellLink from 'common/SpellLink';
 import SpellIcon from 'common/SpellIcon';
```

Comment:
This whole file seems un-needed now. You can easily see this on WCLs

Reply:
@darthelit in my opinion Abe is correct and we should remove. Thoughts?

Reply:
I'm good with removing it. Doesn't add any benefit that can be easily seen in WCL